WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -426,6 +426,24 @@ + (NSDictionary *)processArguments:(XCUIApplication *)app
  */
 + (NSString *)userInterfaceStyle
 {
+#if !TARGET_OS_SIMULATOR
+  // This way is reliable on a real device.
+  NSError *error;
+  long long appearance = [XCUIDevice.sharedDevice fb_getAppearance:&error];
+  if (error == nil) {
+    switch (appearance) {
+      case 0: // UIUserInterfaceStyleUnspecified
```

Comment:
You could use FBUIInterfaceAppearance enum items

Reply:
actually we could avoid some unnecessary duplication since there's a very similar switch below